src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -178,11 +178,19 @@ func (a *App) onBootstrapWorkerNode(ctx context.Context, s state.State, encodedT
 		return fmt.Errorf("failed to write worker node certificates: %w", err)
 	}
 
+	// We don't have the CIDRs on the workers, hence we check the address type of the control-plane node.
```

Comment:
Don't we use the `response.PodCIDR` here later on to setup kubeproxy for workers?

Reply:
yes, wrong assumption. But given that we change the way this is done now, it is obsolete anyway.